src/components/Login/Login.js

```diff
@@ -140,6 +140,7 @@ class Login extends Component {
   }
 
   signInFB() {
+    this.setState({ loaded: false });
```

Comment:
We can write this two lines like this.
```
this.setState(
  { loaded: false },
  () => UserUtil.signInWithFacebook(this.onLoginCallback.bind(this)),
);
```

We can refer this link.
https://facebook.github.io/react/docs/react-component.html#setstate